.github/workflows/deploy.yml

```diff
@@ -0,0 +1,71 @@
+name: Deploy NetKAN-Infra
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    if: ${{ secrets.DOCKERHUB_USERNAME && secrets.DOCKERHUB_PASSWORD }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push NetKAN-infra
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: netkan/.
+          push: true
+          target: production
+          tags: kspckan/netkan:latest
+  webhooks:
+    if: ${{ secrets.DOCKERHUB_USERNAME && secrets.DOCKERHUB_PASSWORD }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+      - name: Build and push webhooks-proxy
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: ./nginx
+          file: ./nginx/Dockerfile
+          push: false
+          tags: kspckan/webhooks-proxy:latest
+
+  deploy:
+    if: ${{ env.AWS_ACCESS_KEY_ID && env.AWS_SECRET_ACCESS_KEY }}
+    runs-on: ubuntu-latest
+    needs: build
```

Comment:
Speaking of concurrency, do we need to add `webhooks` here`? We are restarting the `Webhooks` service below and if I read the `prod-stack.py` correctly, that one also needs the `kspckan/webhooks-proxy` image from the `webhooks` job above.

Reply:
Good catch!